fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/jbatch/bulkdata/export/group/ChunkReader.java

```diff
@@ -140,7 +140,21 @@ public Object readItem() throws Exception {
         pageNum++;
 
         if (chunkData == null) {
-            chunkData = new TransientUserData(pageNum, null, new ArrayList<PartETag>(), 1, 0, null, 0, 0, 0, 1, 0, 1);
+            chunkData = (TransientUserData)TransientUserData.Builder.builder()
+                .pageNum(pageNum)
+                .uploadId(null)
+                .cosDataPacks(new ArrayList<PartETag>())
+                .partNum(1)
+                .indexOfCurrentTypeFilter(0)
+                .resourceTypeSummary(null)
+                .totalResourcesNum(0)
+                .currentUploadResourceNum(0)
+                .currentUploadSize(0)
+                .uploadCount(1)
+                .lastPageNum(0)
+                .lastWritePageNum(1)
+                .build();
+
```

Comment:
I really love this code. Very readable. 

Constants would make this more readable, e.g. why null for resourceTypeSummary

Reply:
This is not something I'm asking to be recoded/changed.  Just a commentary